Documentation/doc/scripts/testsuite.py

```diff
@@ -217,6 +216,14 @@ def main():
             else:
                 sys.stderr.write("Warning: the directory " + publish_dir + dir_to_remove + " does not exist or is not writable!\n")
             revs.eq(k).remove()
+        script_info=d('#suffix')
+        if script_info.text()=='':
+          args_list=''
+          for arg in sys.argv[0:]:
+            args_list+=arg+' '
+          signature='<html><p id="suffix"> Generated with the command line</p><code> python {script_args}</p><p> by {user_name}@{host_name} at {date_time} </code></html>'.format(
+            user_name=getpass.getuser(), host_name=socket.gethostname(), date_time=time.ctime(), script_args=args_list)
+          d('table').after(signature)
```

Comment:
As I read that code: if the `#suffix`  already exists, then it is not updated. It should always be updated.